bitcoin/src/psbt/mod.rs

```diff
@@ -766,7 +766,11 @@ pub enum SignError {
     /// The `scriptPubkey` is not a P2WPKH script.
     NotWpkh,
     /// Sighash computation error.
-    SighashComputation(sighash::Error),
+    Sighash(sighash::Error),
+    /// Taproot sighash computation error.
+    TaprootSighash(sighash::TaprootError),
+    /// P2wpkh sighash computation error.
+    P2wpkhSighash(sighash::P2wpkhError),
```

Comment:
Oh, so why `sighash::Error` doesn't contain both `TaprootError` and `P2wpkhError`?

Reply:
Please see new version, no includes the following:
- `TaprootError`
- `SegwitV0Error`
- `LegacyError`

Eventhough the bottom two have the same fields, I believe this clears up the confusion in `psbt`.